File: app/src/main/java/com/coinninja/coinkeeper/service/client/model/AddressStats.java
package com.coinninja.coinkeeper.service.client.model;

import com.google.gson.annotations.SerializedName;

public class AddressStats {
    String address;
    long balance;
    long received;
    long spent;
    @SerializedName("tx_count")
    int numTransactions;

    public String getAddress() {
        return address;
    }

    public long getBalance() {
        return balance;
    }

    public long getReceived() {
        return received;
    }

    public long getSpent() {
        return spent;
    }

    public int getNumTransactions() {
        return numTransactions;
    }
}

File: app/src/main/java/com/coinninja/coinkeeper/service/client/model/AddressStats.kt
package com.coinninja.coinkeeper.service.client.model

import app.dropbit.annotations.Mockable
import com.google.gson.annotations.SerializedName

@Mockable
data class AddressStats(
        var address: String? = null,
        var balance: Long = 0,
        var received: Long = 0,
        var spent: Long = 0,
        @SerializedName("tx_count")
        var numTransactions: Int = 0
)
